### DISCUSSION TITLE
Have a problem when i try to set up mgen

### DISCUSSION THREAD

**Discussion by LeDucManh251** [USNavalResearchLaboratory/mgen#37] [General]
I have to build MGEN for my project homework of Network Management. I used VirtualBox to do it in Ubuntu. I did exactly the instructions but this error always happen. I dont know how to fix it. Thank you!
![Screenshot from 2022-04-27 00-58-02](https://user-images.githubusercontent.com/92243048/165363724-fec035cb-4fac-4e6d-beaa-d49fd2255be5.png)

**Comment by weston-nrl** [comment]
You're missing the pcap development libraries.  'sudo apt-get install libpcap-dev'
